connexta/eve
# EVE23: Investigate Calendar

#### What does this PR do?
This is a proof of concept for implementing a calendar on the Wallboard using Microsoft Outlook's Calendar.  We used the Microsoft Graph API to pull in the data from a user's calendar and display it using the react-big-calendar library.  In the future, this calendar could be used to display a team's calendar or a company wide calendar.  This implementation is pretty bare bones & rough around the edges, but users are able to log in/log out and have events from their calendar displayed.

#### Who is reviewing it?

@vinamartin @Kjames5269

#### How should this be tested?
You should be able to click the log in button, allow access to your calendar, and see your calendar displayed.  Alex and I both tested this on our accounts.  You can log out of your account afterwards, as well as interact with the calendar.  If you add a new event to your calendar, it should be displayed on the wallboard within 20 seconds.

#### Screenshots
My calendar:
<img width="1920" alt="calendar2" src="https://user-images.githubusercontent.com/20527865/60614133-2fcfcb80-9d81-11e9-9079-f5bd56cd2c57.PNG">

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist.
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
